tests/test_lsmod.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+from drgn import ProgramFlags
+
+import drgn_tools.lsmod as lsmod
+
+
+def test_lsmod(prog):
+    if ProgramFlags.IS_LIVE & prog.flags:
+        return
```

Comment:
What concrete issues have you experienced that required you to skip tests on live kernels?

Our best and quickest tests run on live kernels, it's definitely preferred to run as many tests on the live kernels as possible, unless there's some issue that makes it impractical or infeasible. I can't think of a reason why this wouldn't work, but I may be wrong.

So if there's no reason for this, please delete these lines.